docs/signing-spec.md

```diff
@@ -217,13 +190,51 @@ Verification of an invoice includes the following steps:
 3. Reconstruct the cleartext block following the signing rules
 4. For each signature block
     a. Extract the public key
-    b. Verify the signature using the public key and the cleartext
+    b. Verify that the key has not already been used in another signature block
+        - If a key is used to sign the same invoice multiple times, the implementation SHOULD fail
+    c. Verify the signature using the public key and the cleartext d
         - If verification fails for ANY signature block, fail
-    c. Locate the key in the keyring
+    d. Locate the key in the keyring
         - If the key is not located, this is not an error
-5. Verify that at least one signature block used a key that was present in the keyring (4.c)
-    - If none of the signatures were done with a key we know, fail
+5. Apply a key trust strategy (See "Strategies of Key Trust" below)
+    - At minimum, the strategy should be that the `creator` signature is done with a known key.
```

Comment:
I'd update this to be "At minimum, the strategy should be that the `creator` or `approver` signature is done with a known key."

This more accurately reflects the case outlined above when a bindle can come from an unknown creator but is approved by a known source

Reply:
I updated it to point to the two strategies (Creator is strategy 1, creator or approver is strategy 2) later in the doc.